front/src/views/view-user/view-user.view.js

```diff
@@ -0,0 +1,8 @@
+import { Menu, Footer } from '@layouts';
+
+export default {
+  name: 'User',
+  components: {
+    'ff-menu': Menu, 'ff-footer': Footer
```

Comment:
you could specify correct name in menu and footer components via name property

Reply:
but how to make "export {default as ff-footer}" in index.js?

Reply:
name property in component declaration and name of class gor this component are absolutely different things.

in index.js:
`export {default as Footer}"`

view-user.view.js:
```
import { Menu, Footer } from '@layouts';
export default {
  name: 'User',
  components: {
    Menu, Footer
}
```

user.view.html 
`<ff-footer/>`

footer.layout.js
```
export default {
  name: 'ff-footer'
}
```